.github/CONTRIBUTING.md

```diff
@@ -81,11 +81,10 @@ To get the code from GitHub, you will need to install
 to download the project to your computer.
 
 The full `git clone` command will vary depending on the name of the GitHub
-project in which you are working. Replace `project-template` in the following
-command with the correct name of the project you wish to change:
+project in which you are working. For the "cityzen-server" project the full command is as follows:
```

Comment:
Use the "code quotes" or back-tick character to get code formatting here instead of text quotes ~"cityzeon-server"~. See https://daringfireball.net/projects/markdown/syntax#code for more information about inline code notations in markdown